templates/view-submitted-response.html

```diff
@@ -29,6 +29,17 @@ <h1 class="u-mt-m">
   </h1>
   {{ onsMetadata(content.metadata)}}
   {% if not content.view_submitted_response.expired %}
+    {{
+      onsButton({
+        "type": 'button',
+        "text": 'Print answers',
+        "buttonStyle": "print",
+        "attributes": {
+          "data-qa": "btn-print"
+        }
+      })
+    }}
+    <br><br>
```

Comment:
Instead of break tags, this should be done via CSS, the prototype set `u-mt-l` CSS classes on the summary.